test/spec-urls.test.js

```diff
@@ -54,7 +54,7 @@ describe('spec_url data', () => {
       // Remove if it is in the main ECMA spec
       'https://tc39.es/proposal-hashbang/out.html',
 
-      // Remove if https://github.com/w3c/browser-specs/pull/605 is merged
+      // Remove if https://github.com/w3c/webrtc-extensions/issues/108 is closed
```

Comment:
FYI what will happen here is that the parameter will move to the supported spec. So when that changes we'd remove the exception and change the spec.